ext/node/global.rs

```diff
@@ -467,27 +471,28 @@ pub fn definer<'s>(
   let inner = v8::Local::new(scope, inner);
 
   let Some(success) = inner.define_property(scope, key, descriptor) else {
-    return;
+    return v8::Intercepted::No;
   };
 
   if args.should_throw_on_error() && !success {
     let message = v8::String::new(scope, "Cannot define property").unwrap();
     let exception = v8::Exception::type_error(scope, message);
     scope.throw_exception(exception);
-    return;
+    return v8::Intercepted::No;
   }
 
   rv.set_bool(success);
+  v8::Intercepted::Yes
```

Comment:
Before landing verify if this bool needs to be set or is it used just because of the deprecated V8 API requirements

Reply:
It can be removed it seems:
```diff
- * Use `info.GetReturnValue()` to indicate whether the request was intercepted
- * or not. If the definer successfully intercepts the request, i.e., if the
- * request should not be further executed, call
- * `info.GetReturnValue().Set(value)`. If the definer did not intercept the
- * request, i.e., if the request should be handled as if no interceptor is
- * present, do not not call `Set()` and do not produce side effects.

+ * If the interceptor handles the request (i.e. the property should not be
+ * looked up beyond the interceptor) it should return `Intercepted::kYes`.
+ * If the interceptor does not handle the request it must return
+ * `Intercepted::kNo` and it must not produce side effects.
```

From https://chromium-review.googlesource.com/c/v8/v8/+/5353146